templates/swagger/v1_json.tmpl

```diff
@@ -13683,7 +13683,7 @@
       "x-go-package": "code.gitea.io/gitea/modules/structs"
     },
     "EditOrgOption": {
-      "description": "EditOrgOption options for editing an organization",
+      "description": "EditOrgOption options for editing an organization\nTODO: make optional after https://gitea.com/go-chi/binding/pulls/5 got merged",
```

Comment:
do you really intend this TODO to be in the API?

Reply:
I'll try to move it out of swagger doc ...